wikipron/scrape.py

```diff
@@ -64,13 +54,21 @@ def _scrape_once(data, config: Config) -> Iterator[Pair]:
 def scrape(config: Config) -> Iterator[Pair]:
     """Scrapes with a given configuration."""
     category = _CATEGORY_TEMPLATE.format(language=config.language)
-    next_query = _INITIAL_QUERY_TEMPLATE.format(category=category)
+    requests_params = {
+        "action": "query",
+        "format": "json",
+        "list": "categorymembers",
+        "cmtitle": category,
```

Comment:
I've refactored the GET request logic, per [the MediaWiki API docs](https://www.mediawiki.org/wiki/API:Categorymembers#Examples). This way, we also don't have to worry about the space vs underscore in the language category name, which is nice.